test/demo_web/controllers/user_auth_test.exs

```diff
@@ -63,7 +63,7 @@ defmodule DemoWeb.UserAuthTest do
 
     test "broadcasts to the given live_socket_id", %{conn: conn} do
       live_socket_id = "users_sessions:abcdef-token"
-      Phoenix.PubSub.subscribe(Demo.PubSub, live_socket_id)
```

Comment:
Seeing after this change. 
```
warning: DemoWeb.Endpoint.subscribe/1 is deprecated. DemoWeb.Endpoint.subscribe/2 is deprecated, please call Phoenix.PubSub directly instead
  test/morphic_pro_web/controllers/user_auth_test.exs:66: DemoWeb.UserAuthTest."test logout_user/1 broadcasts to the given live_socket_id"/1
```

Reply:
Correct. The commit message says it requires Phoenix v1.5.2, where those are no longer deprecated. v1.5.2 should be out this week.

Reply:
Ah sorry just seen the commit message.